python/semantic_kernel/skill_definition/sk_function_decorator.py

```diff
@@ -24,6 +26,9 @@ def decorator(func):
         func.__sk_function_name__ = name or func.__name__
         func.__sk_function_input_description__ = input_description or ""
         func.__sk_function_input_default_value__ = input_default_value or ""
+        func.__sk_function_function_completion_enabled__ = (
```

Comment:
Was this intended to be __sk_function_function_.... or just __sk_function_?

Reply:
Indeed, intentional, but not pretty.